examples/rpc/README.md

```diff
@@ -1,7 +1,7 @@
 ## Overview
 
-The `rpc-server` allows  running `ggml` backend on a remote host.
-The RPC backend communicates with one or several instances of `rpc-server` and offloads computations to them.
+`llama-rpc-server` allows  running `ggml` backend on a remote host.
```

Comment:
The `rpc-server` can be used with any `ggml` program and there is nothing llama specific. So I don't think this change is appropriate. Maybe `ggml-rpc-server`?

Reply:
The `llama-` prefix is appropriate here if we're eventually going to have a `llama` supercommand -- it's more about project structure and ownership than specific functionality. 

It may more appropriate for the RPC program to be moved to the ggml project, but as long as it lives here, it feels like it's appropriate to live under the `llama-` supercommand.

Reply:
Yeah, saying that `llama-*` binaries are installed by the `llama.cpp` (or `llama-cpp`) package makes things regular, it's a simpler mental model. `llama-gguf` and `llama-ls-sycl-device` are in the same bucket, the latter not even having anything to do w/ ggml.

Reply:
(@rgerganov otherwise using the ggml branding for everything was briefly considered in https://github.com/ggerganov/llama.cpp/pull/7809#issuecomment-2156509720, I think conceptually it would have been the most logical, but branding has its own logic haha)

Reply:
Ideally the `rpc-server` should be an artifact coming from the `ggml` project but right now it is mixed into `llama.cpp`. The proposed `llama-` naming could be OK for the short term but I can't make the final call here.

Reply:
I agree that `ggml-rpc-server` better reflects the purpose and functionality of the example. It's place is likely better to be in the `ggml` project, but for greater visibility it is currently located in `llama.cpp` so that we can more easily discuss and experiment with potential applications of this new functionality.

On the other hand, I see that it breaks the pattern with the `llama-` prefix. For now, let's keep `rpc-server` as it is and we'll figure out how to improve later

Reply:
SG, reverted to rpc-server.

Good to merge?